alpha/apps/kaltura/lib/storage/kFileSyncUtils.class.php

```diff
@@ -170,16 +172,53 @@ public static function dir_get_files(FileSyncKey $key, $strict = true)
 		return $files;
 	}
 
+	protected static function getCacheKey(FileSyncKey $key)
+	{
+		return self::CACHE_KEY_PREFIX . "{$key->object_id}_{$key->object_type}_{$key->object_sub_type}_{$key->version}";
+	}
+
+	private static function getFromSmallFileCache($cacheKey)
+	{
+		$redisCacheStore = self::initSmallFileRedisInstance();
```

Comment:
Why not use the cache wrapper to do this?
This will make the code much more clean and will allow in the future to change the cache type to whatever is supported by the cache wrappers.